src/components/shared/Button/PaneToggle.css

```diff
@@ -22,10 +22,19 @@
   margin-right: 5px;
 }
 
+html[dir="rtl"] .toggle-button-end {
```

Comment:
You are going in correct direction.

But, I don't think we need to manually add this selector. We use [postcss-bidirection](https://github.com/gasolin/postcss-bidirection) to generate these selectors.

Here's how I would try
```patch
--- src/components/shared/Button/PaneToggle.css
+++ src/components/shared/Button/PaneToggle.css
@@ -23,7 +23,7 @@
 }
 
 .toggle-button-start {
-  margin-left: 5px;
+  margin-inline-start: 5px;
 }
 
 html:not([dir="rtl"]) .toggle-button-end svg,
```

You can use similar approach for `.toggle-button-end`

https://devtools-html.github.io/debugger.html/docs/local-development.html#rtl